components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -122,6 +123,22 @@ class TabsUseCases(
             this.invoke(url, selectTab = true, startLoading = true, parentId = null, flags = flags)
         }
 
+        operator fun invoke(
```

Comment:
Just adding the note so I don't forget. Let's add this to the `SelectOrAddUseCase` below, because that's essentially what it's doing too.